tools/swift-demangle/swift-demangle.cpp

```diff
@@ -127,7 +127,7 @@ static void demangle(llvm::raw_ostream &os, llvm::StringRef name,
 
     if (Classify) {
       std::string Classifications;
-      if (!swift::Demangle::isSwiftSymbol(name.data(), name.size()))
+      if (!swift::Demangle::isSwiftSymbol(name.data()))
```

Comment:
If it didn't matter, I would still ask that you leave a comment and perhaps assert it. But in this case we are taking StringRef slices out of an input stream and trying to demangle them without copying the string data.